<!DOCTYPE html>
<html>
  <head>
    <title>A brave new world of monkeying around with trees – Blog backup – Backup blog posts from various blogs.</title>

        <meta charset="utf-8" />
    <meta content='text/html; charset=utf-8' http-equiv='Content-Type'>
    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
    <meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1.0'>

    
    <meta name="description" content="
I’ve spent the last few days writing an introduction for my first PhD paper on the practical issues of adding fossils to molecular phylogenies (full recipe here).

" />
    <meta property="og:description" content="
I’ve spent the last few days writing an introduction for my first PhD paper on the practical issues of adding fossils to molecular phylogenies (full recipe here).

" />
    
    <meta name="author" content="Blog backup" />

    
    <meta property="og:title" content="A brave new world of monkeying around with trees" />
    <meta property="twitter:title" content="A brave new world of monkeying around with trees" />
    

    <!--[if lt IE 9]>
      <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <link rel="stylesheet" type="text/css" href="/style.css" />
    <link rel="alternate" type="application/rss+xml" title="Blog backup - Backup blog posts from various blogs." href="/feed.xml" />

    <!-- Created with Jekyll Now - http://github.com/barryclark/jekyll-now -->
  </head>

  <body>
    <div class="wrapper-masthead">
      <div class="container">
        <header class="masthead clearfix">
          <a href="/" class="site-avatar"><img src="https://raw.githubusercontent.com/TGuillermeBlog/TGuillermeBlog.github.io/master/images/antlers.jpg" /></a>

          <div class="site-info">
            <h1 class="site-name"><a href="/">Blog backup</a></h1>
            <p class="site-description">Backup blog posts from various blogs.</p>
          </div>

          <nav>
            <!--
            ADDING A PAGE
            <a href="/your_new_page">The page name</a>
            !-->
            <a href="http://tguillerme.github.io/">Website</a>
            <a href="/">Blog</a>
            <a href="/about">About</a>
          </nav>
        </header>
      </div>
    </div>

    <div id="main" role="main" class="container">
      <article class="post">
  <h1>A brave new world of monkeying around with trees</h1>

  <div class="entry">
    <p><img src="/images/Tamarin_portrait_2_edit3-300x200.jpg" alt="headimg" />
I’ve spent the last few days writing an introduction for my first PhD paper on the practical issues of adding fossils to molecular phylogenies (<a href="http://www.youtube.com/watch?feature=player_embedded&amp;v=Y3ciaYSvbyU">full recipe here</a>).</p>

<p>This is my starting point: most people working in macroevolution agree that we should <a href="http://onlinelibrary.wiley.com/doi/10.1111/mee3.2013.4.issue-8/issuetoc">integrate fossils into modern phylogenetic trees</a>. Of the many possible methods that are available, <a href="http://sysbio.oxfordjournals.org/content/61/6/973.short">Ronquist’s total evidence method</a> looks to be the most promising (however, some <a href="http://arxiv.org/abs/1310.2968">nice other</a> ones also exist).</p>

<p>Recently <a href="http://onlinelibrary.wiley.com/doi/10.1111/jeb.12237/abstract">Schrago et al.</a> published a nice attempt to use this method on the Plathyrrini (New-World monkeys to you and me):</p>

<p>As a <a href="http://www.ecoevoblog.com/2013/03/29/dinosaurs-are-useless-if-they-dont-go-in-trees/">reminder</a>, the aim of this total evidence method is to combine all of the available data: both molecular and morphological. Traditionally, analyses have treated each type of data separately; approaches which bring their own advantages and problems.</p>

<p>Let’s start with the molecules:</p>

<p><a href="http://www.sciencedirect.com/science/article/pii/S1055790305003817">Opazo et al.</a> published in 2006 a classical example of a molecular phylogenetics study. There are more recent, impressive phylogenetic studies (like <a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1001342">Perelman et al.</a> in 2011 and <a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0049521">Springer et al.</a> in 2012) on most of the primates and using more genetic data but I think Opazo is a better example of a traditional approach because it involves a tree with 17 taxa instead of more than 200.</p>

<p><img src="/images/Opazo.et_.al2006-Fig5-264x300.png" alt="bodyimg" /></p>
<center>Opazo et al. 2006 Fig. 5. A Platyrrhini dated phylogeny – values indicate the age of the nodes, the circle at the root of the tree is the fossil used for age calibration: Branisella.</center>

<p>Two of the main advantages of this approach are the quantity of data involved (tens of thousands base pairs) and the methods of inferring the evolutionary history: molecular evolutionary models are easy to understand and easy to implement (each site has a finite number of states – A, C, G, T or nothing – and <a href="http://en.wikipedia.org/wiki/Models_of_DNA_evolution">probabilistic models</a> are good enough to infer the rate of changes from one state to the other). From a data perspective, another practical advantage is that, with modern <a href="http://en.wikipedia.org/wiki/DNA_sequencing#Next-generation_methods">NextGen sequencing</a>, it’s really easy and fast to obtain a full genomic dataset. However, the main inconvenience from a macroevolutionary point of view is that molecular approaches don’t really take evidence from the fossil record into account. In the Opazo example, the only fossil used is <a href="http://en.wikipedia.org/wiki/Branisella">Branisella</a>, and the only useful information here is just its age (around 26 Ma) used to calibrate the time on the tree.</p>

<p>On the other hand, <a href="http://www.sciencedirect.com/science/article/pii/S004724840700187X">Kay et al.</a> 2008 published an awesome study of the Platyrrhini history from a palaeontological point of view. They focused on 20 living taxa combined with 11 fossil species and using 268 morphological characters.</p>

<p><img src="/images/Kay.et_.al2008-Fig21-188x300.png" alt="bodyimg" /></p>
<center>Kay et al. 2008 Fig. 21. A Platyrrhini phylogeny based on morphological data including fossils</center>

<p>Again, there are both advantages and problems associated with this approach. Firstly, the number of characters used is pretty low; don’t get me wrong, 268 is really good for a morphological matrix, it’s just low compared to molecular data. Furthermore, the underlying evolutionary models used to build the phylogeny are hard to infer, the most common model is the <a href="http://sysbio.oxfordjournals.org/content/50/6/913.short">Lewis 2001 Mk model</a> where morphological characters are treated as if they “act like” molecular sites with no assumptions made about their states or rates of change (this method has been <a href="http://onlinelibrary.wiley.com/doi/10.1111/cla.12018/full">criticized</a> but it’s still our best way to infer morphological evolution). Another solution, which is also commonly used, is to infer nothing but instead just use a <a href="http://en.wikipedia.org/wiki/Maximum_parsimony_%28phylogenetics%29">maximum parsimony approach</a>: find the tree which explains observed phenotypic evolution with the fewest number of evolutionary steps (characters changing from one state to another on a particular node within the tree). However, compared to a purely molecular approach, the advantages of Kay’s tree are clear from a macroevolutionary point of view: this tree includes full information from the morphologies of both living and fossil species!</p>

<p>Now hopefully you can see where I’m coming from in wanting to use the total evidence method? It’s clear from the empirical examples above that the problems associated with one approach are the advantages in the other. So let’s just combine them! And that’s what Schrago did in their work, they just mixed both data sets and re-ran the analysis (or, more precisely, they used Kay’s data set as it was but added new genomic data collected over the last seven years to Opazo’s data set). Here’s their result:</p>

<p><img src="/images/Schrago.et_.al2006-Fig2-300x236.png" alt="bodyimg" /></p>
<center>Kay et al. 2008 Fig. 21. A Platyrrhini phylogeny based on morphological data including fossils</center>

<p>So here we have the advantage of both methods combined and this tree is far more user friendly for macroevolutionary studies; one can test evolutionary hypothesis through time using a more complete representation of the Platyrrhini evolutionary history. One major problem still remains though; the paucity of useful morphological data compared to the wealth of molecular data which is now available. Does that influence the tree’s topology somehow? Well, stay tuned, my simulations are running…</p>

<p><strong>Photo credit</strong>: wikimedia commons</p>

<p><strong><a href="http://www.ecoevoblog.com/2014/01/13/a-brave-new-world-of-monkeying-around-with-trees/">Original post</a></strong></p>

  </div>

  <div class="date">
    Written on January 13, 2014
  </div>

  
</article>

    </div>

    <div class="wrapper-footer">
      <div class="container">
        <footer class="footer">
          



<a href="https://github.com/TGuillerme/TGuillerme.github.io"><i class="svg-icon github"></i></a>




<a href="https://www.twitter.com/TGuillerme"><i class="svg-icon twitter"></i></a>



        </footer>
      </div>
    </div>

    

  </body>
</html>
